cmd/cmd_test.go

```diff
@@ -90,3 +91,31 @@ func (test *CommandTest) Teardown(t *testing.T) {
 		t.Fatal(err)
 	}
 }
+
+// terminalOverride lets us more easily redirect streams in the tests.
+type terminalOverride struct {
+	oldOut, oldErr, newOut, newErr io.Writer
+}
```

Comment:
This is the new helper that I created, which lets us store off the old `Out` and `Err`, as well as the new ones that we want to override with.

Reply:
This is a good addition for sure. 👍🏽 

 I find the name a bit confusing as it deals more with output redirection that it does terminal overrides. Have you considered going with something along the lines of `CapturedOutput` or `OutputRedirect`?

Reply:
I was really struggling to find a good name. I like both of your suggestions; I think maybe `capturedOutput` is the most expressive.